x-pack/platform/plugins/private/upgrade_assistant/README.md

```diff
@@ -336,8 +336,8 @@ For a complete list of Kibana deprecations, refer to the [8.0 Kibana deprecation
 
 This is a non-exhaustive list of different error scenarios in Upgrade Assistant. It's recommended to use the [tweak browser extension](https://chrome.google.com/webstore/detail/tweak-mock-api-calls/feahianecghpnipmhphmfgmpdodhcapi?hl=en), or something similar, to mock the API calls.
 
-- **Error loading deprecation logging status.** Mock a `404` status code to `GET /api/upgrade_assistant/deprecation_logging`. Alternatively, edit [this line](https://github.com/elastic/kibana/blob/5c13e901ac1bf01bcbcca1d4103c43438ee8f6e6/x-pack/platform/plugins/private/upgrade_assistant/public/application/lib/api.ts#L71) locally and replace `deprecation_logging` with `fake_deprecation_logging`.
-- **Error updating deprecation logging status.** Mock a `404` status code to `PUT /api/upgrade_assistant/deprecation_logging`. Alternatively, edit [this line](https://github.com/elastic/kibana/blob/545c1420c285af8f5eee56f414bd6eca735aea11/x-pack/platform/plugins/private/upgrade_assistant/public/application/lib/api.ts#L77) locally and replace `deprecation_logging` with `fake_deprecation_logging`.
+- **Error loading deprecation logging status.** Mock a `404` status code to `GET /api/upgrade_assistant/deprecation_logging`. Alternatively, edit [this line](https://github.com/elastic/kibana/blob/main/x-pack/platform/plugins/private/upgrade_assistant/public/application/lib/api.ts#L151) locally and replace `deprecation_logging` with `fake_deprecation_logging`.
+- **Error updating deprecation logging status.** Mock a `404` status code to `PUT /api/upgrade_assistant/deprecation_logging`. Alternatively, edit [this line](https://github.com/elastic/kibana/blob/main/x-pack/platform/plugins/private/upgrade_assistant/public/application/lib/api.ts#L141) locally and replace `deprecation_logging` with `fake_deprecation_logging`.
```

Comment:
Can we use permalinks? Code moves, so pointing to `main` will easily become obsolete.